sessions/TypeScript-100/Module-102-common-types/common-types.md

```diff
@@ -0,0 +1,81 @@
+
+# Module 102: Common Types
+
+## Primitives
+
+Along with this file, there are code examples of each type in [common-types-examples.ts](./common-types-examples.ts).
+
```

Comment:
```suggestion

The [TypeScript handbook](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html) includes really good introductory documentation to the "everyday types" that you will encounter on your TS journey! 
```

Reply:
is it worth highlighting how these types aren't the _only_ types important to TS?

Wondering if it's worth alluding to the Utility Types: https://www.typescriptlang.org/docs/handbook/utility-types.html

Reply:
would this be more geared towards a later session potentially? Worth bringing up though, I'll add something in!